packages.json

```diff
@@ -10992,5 +10992,19 @@
     "description": "Nim wrapper for libtess2",
     "license": "MIT",
     "web": "https://github.com/genotrance/nimtess2"
+  },
+  {
+    "name": "witai",
+    "url": "https://github.com/xmonader/witai-nim",
+    "method": "git",
+    "tags": [
+      "wit.ai",
+      "nim",
```

Comment:
The "nim" tag is unnecessary. See https://github.com/nim-lang/packages/issues/399#issuecomment-243021190, https://github.com/nim-lang/packages/pull/886.